### DISCUSSION TITLE
new context suggested for surface water flows

### DISCUSSION THREAD

**Discussion by hottleta** [USEPA/Federal-LCA-Commons-Elementary-Flow-List#141] [Ideas]
To address surface water flows. Previously only available designation is resource/water or emission/water, which is not specific enough because it is inclusive of subterranean flows.

Recommended replacing some existing contexts by adding the surface designation, eg:
-  emission/water/fresh water body
to
- emission/water/surface/fresh water body

Would also need to generate new contexts for:
- resource/water/surface
- emission/water/surface

**Comment by WesIngwersen** [comment]
@ashleyedelen Please review and weigh in here

**Comment by ashleyedelen** [comment]
My understanding was that surface was the unspoken default and that unless subterranean was used then it was by default surface water. This is similar to the concept that we do not have "outdoor" for air emissions. This is the hidden default unless "indoor" is specified. 

I am not opposed to adding this specificity if we decide that it improves the clarity for users, but overall the I believe our current guidance states that unless subterranean is used it is by default surface water.

**Comment by hottleta** [comment]
I don't disagree that it could be implied by the fact that subterranean isn't used, but the designation emission/water is at the highest level and doesn't explicitly rule out subterranean flows which are categorized at the next level. The USGS has some flows that are surface water resources and there are also unspecified water resources but they both end up in the resource/water context.

Thoughts?
